includes/TripalFields/hp__phenotypic_variability/hp__phenotypic_variability.inc

```diff
@@ -182,14 +182,15 @@ class hp__phenotypic_variability extends ChadoField {
     $data = array();
 
     $resource = chado_query(
-      'SELECT trait_id, trait_name, experiment_id, experiment_name, method_id, method_name, unit_id, unit_name 
+      'SELECT trait_id, trait_name, experiment_id, experiment_name, method_id, method_name, unit_id, unit_name
        FROM {mview_phenotype}
        WHERE stock_id = :id
        GROUP BY trait_id, trait_name, experiment_id, experiment_name, method_id, method_name, unit_id, unit_name',
       array(':id' => $germplasm_id));
     while ($r = $resource->fetchObject()) {
       $data[] = array(
         'type' => 'germplasm',
+        'name' => $entity->chado_record->name,
```

Comment:
When adding data to a field which will be shown to the end user, it must be in the value array and it must have a cvterm key. I would suggest adding 
```php
  // Germplasm
  'co_010:germplasm' => array(
    'schema:name' => $entity->chado_record->name
  ),
```

Reply:
I've changed my mind on this one. Since this is only done on the gemplasm page, where the name is already availalble, it's fine to leave it outside the value. Furthermore, we don't want to duplicate it.